guide/ops/starting-stopping-monitoring.md

```diff
@@ -3,81 +3,119 @@ title: Starting, Stopping and Monitoring
 
+Now start the provided brooklyn instance supplying your brooklyn Download credentials in place of the `myuser` and `mypassword` exbrooklynles below
 
-## Platform-independent distributions
+{% highlight bash %}
+$ user=myuser password=mypassword vagrant up brooklyn
+{% endhighlight %}
 
-The platform-independent distributions are packaged in `.tar.gz` and `.zip`
-files.
+If you do not know your username or password (it should have been provided in your welcome email) please contact <a href="mailto:help@cloudsoft.io">help@cloudsoft.io</a> for assistance.
 
+You can check if brooklyn has launched successfully with the command:
 
-### Starting
+{% highlight bash %}
+$ vagrant ssh brooklyn --command 'sudo systemctl status brooklyn'
+{% endhighlight %}
 
-To launch Brooklyn, from the directory where Brooklyn is unpacked, run:
+The application logs can be tailed using:
 
 {% highlight bash %}
-% bin/brooklyn launch > /dev/null 2>&1 & disown
+$ vagrant ssh brooklyn --command 'tail -f /home/vagrant/cloudsoft-brooklyn-karaf/log/brooklyn.info.log'
+$ vagrant ssh brooklyn --command 'tail -f /home/vagrant/cloudsoft-brooklyn-karaf/log/brooklyn.debug.log'
 {% endhighlight %}
 
-With no configuration, this will launch the Brooklyn web console and REST API on [`http://localhost:8081/`](http://localhost:8081/).
-No password is set, but the server is listening only on the loopback network interface for security.
-Once [security is configured](brooklyn_properties.html), Brooklyn will listen on all network interfaces by default.
+</div>
+<div id="impl-2" class="tab-pane fade">
 
-See the [Server CLI Reference](server-cli-reference.html) for more information
-about the Brooklyn server process.
+<strong class="hidden started-pdf-include">Centos / RHEL</strong>
 
-The Brooklyn startup script will create a file name `pid_java` at the root of
-the Brooklyn directory, which contains the PID of the last Brooklyn process to
-be started.
+brooklyn is managed as a system service using the appropriate tool for your system (e.g. systemd). When the brooklyn package is installed, the `brooklyn` service is started for you.
 
+Now, as root (or using sudo), you can monitor brooklyn's status, and stop and start it, with the following commands for systemd:
 
-### Stopping
+{% highlight bash %}
+$ systemctl status brooklyn
+$ systemctl stop brooklyn
+$ systemctl start brooklyn
+{% endhighlight %}
 
-To stop Brooklyn, simply send a `TERM` signal to the Brooklyn process. The PID
-of the most recently run Brooklyn process can be found in the `pid_java` file at
-the root of the Brooklyn directory.
+The application should then output its logs to `/opt/brooklyn/log/brooklyn.info.log` and `/opt/brooklyn/log/brooklyn.debug.log`.
 
-For example:
+Brooklyn will integrate with your OS service management. Commands such as `service brooklyn start` will work as expected, and Brooklyn's PID file will be stored in the normal location for your OS, such as `/var/run/brooklyn.pid`.
```

Comment:
I don't think this will be true (`/var/run/brooklyn.pid`) if using brooklyn karaf. We have more work to do for that; currently it would all be written to `/opt/brooklyn/*` I think.